reference/catalog.v3.yml

```diff
@@ -24148,6 +24514,12 @@ components:
           description: |
             The total quantity of this product sold.
           example: 80
+        channels:
+          type: array
+          description: 'An optional field that can be empty, or you can supply a single ID or an array of IDs.'
```

Comment:
are we 100% that you can actually pass a single ID instead of an array? this seems unlikely.

Reply:
if it is true, then the datatype is a oneOf situation.

Reply:
@slsriehl Kristina explained that it is always an array. The array can have a single ID or multiple IDs in the array.